docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,33 +263,37 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
 . Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
-
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Select **ES|QL**. 
+. Write an aggregating or non-aggregating type query:
```

Comment:
Yeah, I was thinking about making this more general too. Just in case we want to specify more query types in the future. How about something like this:

```suggestion
. Write an ES|QL query. You can create any of the following query types:
```

Reply:
I don't think we need event this: `You can create any of the following query types:`
Just ` Write an ES|QL query`